Ich kriege diesen Fehler:

"
**Failed to compile**

./src/components/SaveProject.js
Module not found: Can't resolve 'axios' in 'C:\Users\Malte\Documents\Medieninformatik\5. Semester\Sourcecode\beuthSoundGroup\Frontend\src\components'
"

Was muss ich tun, um diesen Branch testen zu können?

--- a/Frontend/src/components/multitracknav.js
+++ b/Frontend/src/components/multitracknav.js
@@ -1,0 +1,29 @@
+import React from 'react';
+import { ReactComponent as ArrowForward } from '../img/arrow_forward.svg';
+import { ReactComponent as ArrowBack } from '../img/arrow_back.svg';
+
+
+function MultitrackNav(props) {
+
+    // Functions to be filled with functionality to switch canvas for multitracking
+
+    const prevTrack = () => {
+        console.log("switch to previous track");
+    };
+
+    const nextTrack = () => {
+        console.log("switch to next track");
+    };
+
+
+    return (
+            <div id="multitrackingNav">
+                <ArrowBack id="arrowBack" onClick={() => { prevTrack() }}/>
+                <span id="synthesizerName">synthesizerName</span>
+                <ArrowForward id="arrowForward" onClick={() => { nextTrack() }}/>
+            </div>
+    );
+};
+
+
+export default MultitrackNav;

--- a/Frontend/src/containers/App.js
+++ b/Frontend/src/containers/App.js
@@ -3,6 +3,8 @@ import Menu from '../components/menu'
 import React from 'react';
 import Loopcontrols from "./loopcontrols";
 import SideMenu from '../components/sidemenu';
+import MultitrackNav from '../components/multitracknav';
+
 
 
 function App() {
@@ -10,6 +12,7 @@ function App() {
   return (
     <div className="App">
       <SideMenu />
+      <MultitrackNav />
       <Loopcontrols />
       <Canvas />
       <Menu />

--- a/Frontend/src/img/arrow_back.svg
+++ b/Frontend/src/img/arrow_back.svg
@@ -1,0 +1,1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0z" fill="none"/><path fill="#282828" d="M11.67 3.87L9.9 2.1 0 12l9.9 9.9 1.77-1.77L3.54 12z"/></svg>

--- a/Frontend/src/img/arrow_forward.svg
+++ b/Frontend/src/img/arrow_forward.svg
@@ -1,0 +1,1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0z" fill="none"/><path fill="#282828" d="M5.88 4.12L13.76 12l-7.88 7.88L8 22l10-10L8 2z"/></svg>

--- a/Frontend/src/index.css
+++ b/Frontend/src/index.css
@@ -488,3 +488,34 @@ body {
   padding: 0 0.2vw;
   color: #a5a4a4;
 }
+
+/*********** Navigation Multitracking **************/
+
+#multitrackingNav {
+  position: absolute;
+  left: 0;
+  right: 0;
+  text-align: center;
+  margin-top: 2vw;
+  height: 0;
+}
+
+#arrowForward {
+  width: 2.2vw;
+  height: 2.2vw;
+  cursor: pointer;
+}
+
+#arrowBack {
+  width: 2.2vw;
+  height: 2.2vw;
+  cursor: pointer;
+}
+
+#synthesizerName {
+  font-size: 1.5vw;
+  display: inline-block;
+  vertical-align: middle;
+  margin: -1.6vw 0.8vw 0 0.3vw;
+  user-select: none;
+}

